sections/main-product.liquid

```diff
@@ -205,9 +205,14 @@
               {{- block.settings.text -}}
```

Comment:
We're not hiding the text block at the moment but I think maybe we should ? 
![](https://screenshot.click/21-05-g06jo-a9ilb.png)

Reply:
Because the product vendor element (`OS 2.0`) or the product subtitle are simply a `Text` block with a linked metafield, I kept this type of block available on modals. I would personally vote to keep as it can also be used by merchants to add important information to the product's template. 

However, @duygukalaycioglu can confirm whether or not it should be part of the modal.

Reply:
After meeting up we came to the conclusion that we'd keep the text block in the modal 👍